app/parse.py

```diff
@@ -1,5 +1,14 @@
 from dataclasses import dataclass
 from enum import Enum
+from urllib.parse import urljoin
+
+import requests
+from bs4 import BeautifulSoup, Tag
+
+BASE_URL = "https://mate.academy/"
+HOME_URL = urljoin(BASE_URL, "en")
+PAGE = requests.get(HOME_URL).content
+SOUP = BeautifulSoup(PAGE, "html.parser")
```

Comment:
It's not very good to use constants for these, make constant only BASE_URL, and other variables move to the function